fixtures/workers-with-assets-run-worker-first/tests/index.test.ts

```diff
@@ -3,35 +3,43 @@ import { fetch } from "undici";
 import { afterAll, beforeAll, describe, it } from "vitest";
 import { runWranglerDev } from "../../shared/src/run-wrangler-long-lived";
 
-describe("[Workers + Assets] run_worker_first true", () => {
-	let ip: string, port: number, stop: (() => Promise<unknown>) | undefined;
+const devCmds = [{ args: [] }, { args: ["--x-assets-rpc"] }];
 
-	beforeAll(async () => {
-		({ ip, port, stop } = await runWranglerDev(resolve(__dirname, ".."), [
-			"--port=0",
-			"--inspector-port=0",
-		]));
-	});
+describe.each(devCmds)(
```

Comment:
no tests changes were added here!! We are basically running the existing set of tests for both `wrangler dev` and `wrangler dev --x-assets-rpc`